spec/schemas/region.yaml

```diff
@@ -53,3 +60,5 @@ Provider:
     url:
       type: string
       example: "https://demo.secapi.cloud/providers/seca.network"
+  description: |
+    A provider of cloud services
```

Comment:
put the description after the type, at the top and not sometimes and the end and sometimes at the top, always at the top after the type (if a type is present)